src/notify/subscription/db.go

```diff
@@ -42,13 +37,12 @@ func get(id string) (Subscription, error) {
 	defer rows.Close()
 
 	if rows.Next() {
-		err := scan(rows, &subscription)
-		if err != nil {
+		if err := scan(rows, &subscription); err != nil {
 			return subscription, err
 		}
 	}
 
-	return subscription, nil
+	return subscription, errors.New("No subscription found")
```

Comment:
OK - this will all cleanup with QueryRow(), but looking at it again, I don't think you should be returning error.New() at the end - that's not an error case - that's success.